addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
setting scrollTop is synchronous. The scroll event caused by setting it is fired off almost immediately, but you would need to wait for it.  It would be captures by any `setTimeout` based waiter though, which already exists iirc.